lte/gateway/python/magma/pipelined/tunnel_id_store.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import threading
+
+from magma.common.redis.client import get_default_client
+from magma.common.redis.containers import RedisHashDict
+from magma.common.redis.serializers import get_json_deserializer, \
+    get_json_serializer
+
+
+class TunnelToTunnelMapper:
+    """
+    Interface ID to Number Mapper
+
+    This class maintains dictionary from tunnel id to tunnel id
+    """
+
+    def __init__(self):
+        self._tunnel_map = TunnelDict()
+        self._lock = threading.Lock()  # write lock
+
+    def get_tunnel(self, tunnel):
+        with self._lock:
+            if tunnel not in self._tunnel_map:
+                return None
+            return self._tunnel_map[tunnel]
+
+    def save_tunnels(self, uplink_tunnel: str, downlink_tunnel: str):
+        with self._lock:
+            self._tunnel_map[uplink_tunnel] = downlink_tunnel
+            self._tunnel_map[downlink_tunnel] = uplink_tunnel
+
+
```

Comment:
I think we can get rid of redis map, this info can be encoded in OVS register. controller read it from ovs flow msg.
do you see any issue with that?

Reply:
discussed offline, will take a task for reanalyzing the redis related stuff once 5g work is done